data/mt/retired/Mary-Caferro-c2eda1fb-6dc5-4326-a4b5-a295f26ee399.yml

```diff
@@ -3,20 +3,26 @@ name: Mary Caferro
 given_name: Mary
 family_name: Caferro
 gender: Female
-email: marycaferro@gmail.com
+email: Mary.Caferro@legmt.gov
 party:
 - name: Democratic
 roles:
 - end_date: 2018-12-31
   type: upper
   jurisdiction: ocd-jurisdiction/country:us/state:mt/government
   district: '41'
-- type: lower
+- end_date: '2022-12-31'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:mt/government
   district: '81'
+- end_date: '2022-12-31'
```

Comment:
@bfossen-ce this person served in the house (district 81) from 2019 until 01/02/2023. On that date they are starting in the house district 82